packages/fluentui/react-northstar/src/components/Table/Table.tsx

```diff
@@ -58,7 +58,7 @@ export const tableSlotClassNames: TableSlotClassNames = {
   header: `${tableClassName}__header`,
 };
 
-export type TableStylesProps = never
+export type TableStylesProps = never;
```

Comment:
This change seems unrelated

Reply:
auto generated after taking latest pull and running Yarn commands